annoyed-alligators/socl_media/apps/feed/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.urls import reverse
+from PIL import Image
+
+
+class Post(models.Model):
+    """
+    The model to store Posts created by users.
+    Fields-
+    post_content: The text of the post
+    posted_by: User who created the post
+    post_date_posted: Date of making the post
+    post_image: Image posted with the post
+    """
+
+    post_content = models.TextField(verbose_name="content")
+    posted_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    post_date_posted = models.DateTimeField(auto_now_add=True,
+                                            verbose_name="Published on")
+    post_image = models.ImageField(upload_to="post_images", null=True, blank=True)
+
+    def get_absolute_url(self):
+        return reverse('post-detail', kwargs={'pk': self.pk})
+
+    def save(self, **kwargs):
+        """
+        This function will resize and add a pixelating effect to the images
+        to give them a more retro and early internet feel.
+        """
+        super().save()
```

Comment:
First things that comes to mind is that *args and **kwargs might be useful if you ever want to use `super`